scripts/teams-bulk-create-teams/README.md

```diff
@@ -79,6 +79,67 @@ end{
 
 [!INCLUDE [More about PnP PowerShell](../../docfx/includes/MORE-PNPPS.md)]
 
+# [CLI for Microsoft 365](#tab/cli-m365-ps)
+```powershell
+# Usage example:
+# .\Create-BulkTeams.ps1 -OwnerEmail "user@contoso.com"
+
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory = $true)]
+    [string]$OwnerEmail,
+    [Parameter(Mandatory = $false)]
+    [string] $SiteListJsonFile = ".\teams.json"
+)
+begin {
+    #Log in to Microsoft 365
+    Write-Host "Connecting to Tenant" -f Yellow
+
+    $m365Status = m365 status
+    if ($m365Status -match "Logged Out") {
+        m365 login
+    }
+
+    Write-Host "Connection Successful!" -f Green
+
+    $sites = Get-Content $SiteListJsonFile -Raw | ConvertFrom-Json
+    $prefixes = "HR", "Finance", "ICT", "Service Desk", "Client Services", "Project Alpha", "Project Beta", "Project Charlie", "Leadership", "Community"
+}
+process {
+    $prefixes | Foreach-Object {
+        $prefix = $_
+
+        $sites | Foreach-Object {
+            $siteTitle = "$($_.SiteTitle.Replace("XXXXX", $prefix))"
+            $mailNickname = "$($_.SiteUrl.Replace("XXXXX", $prefix).Replace(" ",''))"
+
+            # Check if team exists
+            $team = m365 teams team get --name $siteTitle | ConvertFrom-Json
+
+            if (-not $team) {
+                # Create a Microsoft 365 Group
+                Write-Host "Creating group $($siteTitle)"
+                m365 aad o365group add --displayName "$($siteTitle)" --description "Testing Site for $($siteTitle)" --mailNickname $mailNickname --owners "$($OwnerEmail)" --isPrivate (!$_.IsPublic).ToString().ToLowerInvariant()
+
```

Comment:
we could add a remark here (in a comment above this command) that if the `$OwnerEmail` will be the tenant admin than this will break 😉.
This is a very special case were when we create a new o365group the tenant admin will be by default the member of this group internally. Then when we want to add  the same user as the owner of this group we will get the following error
`Error: One or more added object references already exist for the following modified properties: 'owners'.`

Reply:
Thanks. I will add this remark.